### PR TITLE
Fixes in update-alternatives chapter

### PR DESCRIPTION
**Pull request by sknorr**
Grammar fixes + style checker adjustments (mostly "man pages")

### CODE REVIEW

File: xml/update-alternatives.xml [hunk 1]
```diff
@@ -84,7 +83,7 @@
      <term>Link group</term>
      <listitem>
       <para>
-       A set of related symlinks that can be to be updated as a group.
+       A set of related symbolic links that can be updated as a group.
```
Comment (tomschr) [inline xml/update-alternatives.xml]: I have a problem with this change. Although it's technically correct, the manpage talks about symlinks, not symbolic links. Additionally, the term is definied in the terminology list. I would suggest to phrase it like this:

```
A set of related symlinks (symbolic links) that ...
```

File: xml/update-alternatives.xml [hunk 2]
```diff
@@ -106,30 +105,36 @@
...
     </varlistentry>
     <varlistentry>
-     <term>Symlink</term>
+     <term>Symbolic link</term>
      <listitem>
       <para>
-       A symbolic link in the alternative directory.
-       This is the link that can be modified by the administrator through
-       the <command>update-alternatives</command> command.
+       A file that is reference to another file in the same file system.
```
Comment (sknorr) [inline xml/update-alternatives.xml]: forgot sth: "...is _a_ reference..."

File: xml/update-alternatives.xml [hunk 3]
```diff
@@ -301,8 +309,8 @@ OpenJDK 64-Bit Server VM (build 24.141-b02, mixed mode)</screen>
    <itemizedlist>
     <listitem>
      <para>
-      When switching from auto to manual mode and installing another Java
-      version, the &as; does not touch the links nor it changes the
+      When switching from automatic to manual mode and installing another Java
+      version, the &as; does neither touches the links nor changes the
```
Comment (sknorr) [inline xml/update-alternatives.xml]: remove "does"

File: xml/update-alternatives.xml [hunk 4]
```diff
@@ -323,14 +331,15 @@ OpenJDK 64-Bit Server VM (build 24.141-b02, mixed mode)</screen>
   </sect1>
 
   <sect1 xml:id="sec.ua.install">
-   <title>Installing your Custom Alternatives</title>
+   <title>Installing a Custom Alternatives</title>
```
Comment (sknorr) [inline xml/update-alternatives.xml]: this is an issue of my own making -- maybe just remove "a" here.

File: xml/update-alternatives.xml [hunk 5]
```diff
@@ -428,38 +437,39 @@ foo - auto mode
...
 
-  <sect1 xml:id="sec.ua.slaves">
+  <sect1 xml:id="sec.ua.slave">
    <title>Defining Dependent Alternatives</title>
    <para>
-    If you have alternatives, the script itself is not enough. Usually
-    scripts comes with a manpage or some other configuration options.
-    To create alternatives which are dependent from a master
-    link you need to create <emphasis>slave alternatives</emphasis>.
+    If you have alternatives, the script itself is not enough. Most commands
```
Comment (sknorr) [inline xml/update-alternatives.xml]: "the script itself is not enough" sounds weird, as if this was an inevitable thing though it isn't. maybe: "Setting up alternatives for a script is often not enough."

File: xml/update-alternatives.xml [hunk 6]
```diff
@@ -106,30 +105,36 @@
...
     applicable to other file types.
    </para>
    <para>
     The &as; uses the alternatives directory to collect links to possible
     alternatives. When a new package with an alternative is installed,
-    the new alternative is added to the system. How it is integrated
-    depends on the priority and on the mode. Usually, packages
-    with a higher version have also a higher priority. The &as; can operate
+    the new alternative is added to the system. Whether the new package's
+    alternative is selected as the default, depends its priority and on the
```
Comment (tomschr) [inline xml/update-alternatives.xml]: Should it be "depends on"...?

**Comment by tomschr** [comment]
Ok, I've fixed this and merged it to develop. Many thanks! 👍

**Pull request closed without merging**